packages/itwinui-css/src/anchor/anchor.scss

```diff
@@ -1,9 +1,61 @@
 // Copyright (c) Bentley Systems, Incorporated. All rights reserved.
 // See LICENSE.md in the project root for license terms and full copyright notice.
-@use './mixins' as *;
+@use '../mixins';
 
 .iui-anchor {
-  @include iui-anchor;
+  &:where(button) {
+    font: inherit;
+    border: none;
+    background: transparent;
+    padding: 0;
+  }
+
+  @include mixins.iui-focus($color: var(--iui-color-text-accent), $offset: 1px);
+  color: var(--iui-color-text-accent);
+  -webkit-tap-highlight-color: hsl(var(--iui-color-accent-hsl) / var(--iui-opacity-6));
+
+  &:hover {
+    color: var(--iui-color-text-accent-hover);
+  }
+
+  @each $statusColor in informational, positive, warning, negative {
+    &:where([data-iui-status='#{$statusColor}']) {
+      @include mixins.iui-focus($color: var(--iui-color-text-#{$statusColor}), $offset: 1px);
+      color: var(--iui-color-text-#{$statusColor});
+      -webkit-tap-highlight-color: hsl(var(--iui-color-#{$statusColor}-hsl) / var(--iui-opacity-6));
+      @include mixins.iui-text-selection(#{$statusColor});
+
+      &:hover {
+        color: var(--iui-color-text-#{$statusColor}-hover);
+      }
+    }
+  }
+
+  border-radius: var(--iui-border-radius-1);
+  cursor: pointer;
+
+  text-decoration: none;
```

Comment:
Nit: could you reorder this file so that these loose declarations (`color`, `cursor`, etc) are at the top of the class, and the `:hover` declarations are grouped, and the status colors are at the bottom?

Reply:
Sure, will do.

Reply:
Done in this [commit.](https://github.com/iTwin/iTwinUI/pull/1915/commits/aae66014d51e43dc6228a8bb087f946daf705f44)